static/app/types/metrics.tsx

```diff
@@ -0,0 +1,8 @@
+export type MetricTag = {
+  key: string;
+};
+
+export type MetricTagValue = {
+  key: string;
+  value: string;
+};
```

Comment:
I like the separate module for types and think we should do more of that. Should `metrics/index` import and re-export these types as well so that consumers of types only have one path to import from?